pkg/testcase/pod.go

```diff
@@ -23,15 +24,24 @@ func TestPodStatus(
 	podAssertRestarts,
 	podAssertReady assert.PodAssertFunc,
 ) {
-	Eventually(func(g Gomega) {
+	cmd := "kubectl get pods -A --field-selector=status.phase!=Running | " +
+		"kubectl get pods -A --field-selector=status.phase=Pending"
+	Eventually(func(g Gomega) bool {
 		pods, err := shared.GetPods(false)
 		g.Expect(err).NotTo(HaveOccurred())
 		g.Expect(pods).NotTo(BeEmpty())
 
+		res, _ := shared.RunCommandHost(cmd + " --kubeconfig=" + shared.KubeConfigFile)
+		if res != "" {
+			shared.LogLevel("info", "Pods not Running or Pending: \n%s", res)
```

Comment:
Waiting for the pod status to become "Running".

Reply:
Updated